platform/ios/app/MBXAppDelegate.m

```diff
@@ -15,6 +15,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 #endif
 
     [MGLMetricsManager sharedManager].delegate = self;
+
+    // Temporary key created just for this hack
+    [MGLAccountManager setAccessToken:@"pk.eyJ1IjoidG1wc2FudG9zIiwiYSI6ImNqeXY0MTJ5ejBrNHMzYm40eTBtaTc5YncifQ.jb6YkzCG4QpJon7mdc4giA"];
```

Comment:
nit: don’t forget to rotate/drop this token

Reply:
@frederoni the name of this token on my account is "PLEASE DELETE ME" :-D